test/Parameters.js

```diff
@@ -74,34 +80,52 @@ describe('Parameters contract Tests', async () => {
 
   it('parameters should not be modified without admin role access', async () => {
     let tx = parameters.connect(signers[1]).setPenaltyNotRevealNum(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setPenaltyNotRevealDeom(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
+
+    tx = parameters.connect(signers[1]).setSlashPenaltyNum(toBigNumber('1'));
+    assertRevert(tx, expectedRevertMessage);
+
+    tx = parameters.connect(signers[1]).setSlashPenaltyDenom(toBigNumber('1'));
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setWithdrawLockPeriod(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
+
+    tx = parameters.connect(signers[1]).setWithdrawReleasePeriod(toBigNumber('1'));
+    assertRevert(tx, expectedRevertMessage);
+
+    tx = parameters.connect(signers[1]).setResetLockPenalty(toBigNumber('1'));
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setMaxAltBlocks(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setEpochLength(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setNumStates(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setExposureDenominator(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setMinStake(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setGracePeriod(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
 
     tx = parameters.connect(signers[1]).setAggregationRange(toBigNumber('1'));
-    await assertRevert(tx, expectedRevertMessage);
+    assertRevert(tx, expectedRevertMessage);
+
+    tx = parameters.connect(signers[1]).setmaxAssetsPerStaker(toBigNumber('1'));
```

Comment:
Hey Hi @GauravJain9  
we have removed this from the latest master 
Please update your branch and this testcase as well

Reply:
Yeah I did now @abhishekvispute